build/Versions.props

```diff
@@ -67,6 +67,7 @@
     <!-- Roslyn -->
     <MicrosoftVisualStudioLanguageServicesVersion>2.6.0-beta1-62113-02</MicrosoftVisualStudioLanguageServicesVersion>
     <MicrosoftVisualStudioIntegrationTestUtilitiesVersion>2.6.0-beta1-62113-02</MicrosoftVisualStudioIntegrationTestUtilitiesVersion>
+    <MicrosoftCodeAnalysisPooledObjectsVersion>2.6.0-beta1-62113-02</MicrosoftCodeAnalysisPooledObjectsVersion>
```

Comment:
Could you use a newer version? This version does not have source link support, which might result in build failures. 
2.7.0-beta3-62506-01 should be good (even newer ones require some toolset update that's not ready yet).

Reply:
I'll have a new RepoToolset built in a moment that supports the latest compiler source packages.

Reply:
https://github.com/dotnet/project-system/pull/3135

Reply:
Yeah, I was just trying to match the version of the rest of the Roslyn packages, but I can update this one with a comment about why it is different.  So I should pick the latest 2.7 package, right?

Reply:
Updated, but I'm seeing a new error on my work machine that I wasn't seeing at home:

> ProjectSystem\VS\Tree\Dependencies\Snapshot\Dependency.cs(337,23): error CS0122: 'PooledStringBuilder' is inaccessible due to its protection level [C:\Code\project-system\src\Microsoft.VisualStudio.ProjectSystem.Managed.VS\mvsokklc.tmp_proj]

Looks like the xaml tmp_proj isn't getting the files added to it.  Any ideas?

Reply:
Odd. Not sure how much the XAML project differs from regular project. My guess would be that nuget can't determine what language it is from the extension and thus doesn't know what language to pick from the source package content types. Perhaps forcing ```<Language>C#</Language>``` in Directory.Build.props file next to the project might work?